Lib/sqlite3/__main__.py

```diff
@@ -25,22 +25,28 @@ def execute(c, sql, suppress_errors=True):
         for row in c.execute(sql):
             print(row)
     except sqlite3.Error as e:
+        t = theme.traceback
         tp = type(e).__name__
         try:
-            print(f"{tp} ({e.sqlite_errorname}): {e}", file=sys.stderr)
+            tp += f" ({e.sqlite_errorname})"
         except AttributeError:
-            print(f"{tp}: {e}", file=sys.stderr)
+            pass
+        print(
+            f"{t.type}{tp}{t.reset}: {t.message}{e}{t.reset}",
+            file=sys.stderr,
+        )
         if not suppress_errors:
             sys.exit(1)
 
 
 class SqliteInteractiveConsole(InteractiveConsole):
     """A simple SQLite REPL."""
 
-    def __init__(self, connection):
+    def __init__(self, connection, use_color=False):
```

Comment:
As I understand it, you have this here so that somebody can import `sqlite3.__main__` (even if that's a weird import to do) and force for there not be colors. The `False` default here ensures that if somebody's already doing that, they won't be surprised with ANSI escape codes in their output.